code/modules/projectiles/guns/projectile/automatic.dm

```diff
@@ -88,6 +88,9 @@
 	burst_size = 2
 	pin = /obj/item/device/firing_pin/implant/pindicate
 
+/obj/item/weapon/gun/projectile/automatic/c20r/unrestricted
+	pin = /obj/item/device/firing_pin

```

Comment:
My mistake, it is done in places now that I mention it, but not here.
The point I was making erroneously was just constant expressions.

In any case, what Ikky has done here is fine.